dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/DefaultClients.java

```diff
@@ -164,4 +191,24 @@ private static Request accepting(Request original, String acceptValue) {
                 .putHeaderParams(HttpHeaders.ACCEPT, acceptValue)
                 .build();
     }
+
+    private static final class EndpointChannelAdapter implements EndpointChannel {
```

Comment:
duplicate class?

Reply:
Not quite, one is in :serde and the other is in :core, we don't have a dep in either direction. I named em the same just because they're identical